backend/app/treasury/storage_helpers/storage_helpers.py

```diff
@@ -0,0 +1,26 @@
+import json
```

Comment:
Same remark as in backend/app/token_whitelists/storage_helpers/storage_helpers.py